### ISSUE TITLE
Bug: error display limits under deploy page when pod hav multiple containers.

### ISSUE THREAD

**Issue by zzjin** [aptakube/aptakube#200]
### Current and expected behavior

When open deployment page with one, the pods are shown as missing limits like below:
![image](https://github.com/aptakube/aptakube/assets/679757/4935efac-be0a-451a-86bc-fd03447e0d66)
But click into detail pod pages, we can see that first pod have limit:
![image](https://github.com/aptakube/aptakube/assets/679757/44e7f0c0-17f4-4f92-b3db-199584dbf65d)
So there must be some misleading.

### Environment & Operating System

Latest

**Comment by goenning** [comment]
Hi @zzjin 

that’s actually working as designed because your second container doesn’t have limits defined, so technically the pod doesn’t have limit as the second container may use as much cpu/memory it needs.

Have you considered adding some limits to those sidecars too?

**Comment by zzjin** [comment]
That's controlled by some other operator and thus cannot just edit pods's yaml to take effect.

Can aptakube show some detailed info about usage when hover on?
```
181% of requested 500m
30% of limit 3000m
1/2 containers are missing limit
```
something like this.
The real problem(misleading) I found here is, that the first container runs over it's limits(when take screenshot it's down however), but I see there is no limits on the deployment page at first glance. As the deploy is not down by myself, I do not known there have one container hang at first time.

**Comment by goenning** [comment]
I agree this could be better. It's quite a tricky scenario, but I'll see what I can do. 
Probably something like what you suggested, but there are some edge cases.

Like in your screenshot above, if the usage on the second container was 2500m, the total Pod CPU usage would be 1000m+2500m, while the limit is only 3000m as the second container doesn't have one.

Showing `116% of limit 3000m` would be confusing.

Maybe I could split this popup by container, so it'd show like:

```
minio:
181% of requested 500m
30% of limit 3000m
---
sidecar:
Usage: 1m
Missing Requests/Limits
```

**Comment by zzjin** [comment]
Sounds good, but when containers gets more(we have some pods each have 6 containers),the display will be catastrophic.

**Comment by goenning** [comment]
6 normal containers or some are init?

It’s unusual to have that many 😅

**Comment by zzjin** [comment]
yeah it's not common, just forget it.
